src/web/index.js

```diff
@@ -166,6 +166,12 @@ window.addEventListener("message", ({ source, data: message }) => {
   }
 });
 
+document.addEventListener("click", event => {
+  if (event.target.title === "Revert All") {
```

Comment:
This is quite indirect. Why aren't you exposing a helper function through `props` via https://github.com/mozilla/FirefoxColor/blob/a43d5c66c7dc6f644f4e42e6a53ed1df9e19f099/src/web/index.js#L200-L211 ?

Reply:
I wasn't sure about this but I wanted to use the postMessage so the background.js would be aware. I'll revisit this though..